src/platform_impl/ios/window.rs

```diff
@@ -58,7 +58,11 @@ impl Inner {
 
     pub fn request_redraw(&self) {
         unsafe {
-            let () = msg_send![self.view, setNeedsDisplay];
+            if self.gl_or_metal_backed {
```

Comment:
This definitely needs a comment. [`setNeedsDisplay`](https://developer.apple.com/documentation/uikit/uiview/1622437-setneedsdisplay?language=objc) does not work with gl backed views. I'll double check metal since it's not documented.

Reply:
Added some comments, and verified CAMetalLayer backed views don't respond to `setNeedsDisplay`.